### PR TITLE
[website] Update link to environment_type (SDK harness configuration)

### PR DESCRIPTION
**Pull request by iemejia**
R: @ibzib

### CODE REVIEW

File: website/src/roadmap/portability.md [hunk 1]
```diff
@@ -165,7 +165,3 @@ Please see the [Spark Runner page]({{ site.baseurl }}/documentation/runners/spar
 how to run portable pipelines on top of Spark.
 
 Python streaming mode is not yet supported on Spark.
-
-## SDK Harness Configuration {#sdk-harness-config}
```
Comment (ibzib) [inline website/src/roadmap/portability.md]: Can we leave this?
a) Links to this header could be left floating around out there in my emails etc.
b) I also think it's useful to link these two pages.

Comment (iemejia) [reply website/src/roadmap/portability.md]: Yes makes sense, will rollback that change.

**Comment by iemejia** [comment]
Suggested fixes done PTAL @ibzib

**Comment by iemejia** [comment]
Thanks Kyle!

**Pull request merged**